net/HttpRequest.hpp

```diff
@@ -1428,7 +1476,7 @@ class Session final : public ProtocolHandlerInterface
                 }
             }
 
-            if (_response->get("Connection", "") == "close")
+            if( _response->header().getConnectionToken() == Header::ConnectionToken::Close )
```

Comment:
Somehow the space between the `if` and the paren went away. `if`, `for`, `while`, etc. are not functions. It's customary to leave a space to differentiate between them and function calls.

(Again, making it easier for maintainers to read and parse code quickly and will lower chance of confusion/misreading is a good thing.)